core/src/main/java/pl/agh/edu/ui/panel/DifficultyPanel.java

```diff
@@ -0,0 +1,143 @@
+package pl.agh.edu.ui.panel;
+
+import static pl.agh.edu.ui.utils.SkinFont.H1;
+import static pl.agh.edu.ui.utils.SkinFont.H2;
+import static pl.agh.edu.ui.utils.SkinFont.H3;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.badlogic.gdx.scenes.scene2d.ui.ButtonGroup;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.NinePatchDrawable;
+
+import pl.agh.edu.config.GraphicConfig;
+import pl.agh.edu.engine.hotel.dificulty.DifficultyLevel;
+import pl.agh.edu.ui.GameSkin;
+import pl.agh.edu.ui.component.button.DifficultyButton;
+import pl.agh.edu.ui.component.button.ScenarioLabeledButton;
+import pl.agh.edu.ui.component.label.LanguageLabel;
+import pl.agh.edu.ui.resolution.Size;
+import pl.agh.edu.ui.utils.SkinColor;
+import pl.agh.edu.ui.utils.wrapper.WrapperContainer;
+import pl.agh.edu.utils.LanguageString;
+
+public class DifficultyPanel extends WrapperContainer<Table> {
+	public final GameSkin skin = GameSkin.getInstance();
+	public final Table frame = new Table();
+	public final List<DifficultyButton> buttonsList = new ArrayList<>();
+	public final ButtonGroup<TextButton> buttonGroup = new ButtonGroup<>();
+	private LanguageLabel titleLabel;
+	private ScenarioLabeledButton backButton;
+	private ScenarioLabeledButton playButton;
+	private float largePaddingMultiplier = 1;
+
+	public DifficultyPanel() {
+		setActor(frame);
+		getSize();
+		createDifficultyButtons();
+		createTitleLabel();
+		createPlayButton();
+		createBackButton();
+		createFrame();
+		setResolutionChangeHandler(this::updateSizes);
+	}
+
+	public void createFrame() {
+		frame.clearChildren();
+		frame.left().top();
+		frame.setFillParent(true);
+		frame.background(skin.getDrawable("hotel-room"));
+		frame.add(createTitleLabelTable()).left().padLeft(frame.getWidth() / 12).padTop(largePaddingMultiplier * frame.getHeight() / 16).expandX().row();
+		addDifficultyButtonsToFrame();
+		addPlayBackButtonsToFrame();
+	}
+
+	private Table createTitleLabelTable() {
+		Table titleLabelTable = new Table();
+		titleLabelTable.setBackground(getTitleLabelBackground());
+		titleLabelTable.add(titleLabel).pad(10f, 40f, 10f, 40f);
+		return titleLabelTable;
+	}
+
+	private void addPlayBackButtonsToFrame() {
+		Table playBack = new Table();
+		playBack.add(backButton).padRight(frame.getWidth() / 2);
+		playBack.add(playButton);
+		frame.add(playBack).padTop(largePaddingMultiplier * frame.getHeight() / 15);
+	}
+
+	private void addDifficultyButtonsToFrame() {
+		buttonsList.forEach(button -> frame.add(button).left().padTop((int) (largePaddingMultiplier * frame.getHeight() / 24)).padLeft(frame.getWidth() / 6 + buttonsList.indexOf(
```

Comment:
z tymi wszystkimi wartościami bym wyciągnał do statycznej klasy i ograł w niej `largePaddingMultiplier` w zależności od size trochę czytelniej i cała metodologia tam zostaje ale up to you